test/pylib/manager_client.py

```
@@ -47,13 +48,16 @@ class ManagerClient:
         cluster_manager (ScyllaClusterManager): the manager to drive
         manager_loop: the event loop the manager runs on
         con_gen (Callable): generator function for CQL driver connection to a cluster
+        mode (str): test.py build mode, for scaling timeouts to match the manager's
     """
     # pylint: disable=too-many-public-methods
 
     def __init__(self, cluster_manager: ScyllaClusterManager, manager_loop: asyncio.AbstractEventLoop,
                  port: int, use_ssl: bool, auth_provider: Any|None,
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)